src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,28 @@
+import React, {Component} from 'react';
```

Comment:
If you are just creating a functional component, you don't have to import `{Component}`